locales/en/index.yml

```diff
@@ -2457,17 +2459,19 @@ features:
         title: "TMP"
         subtitle: "TMP"
       notFound:
-        title: "TMP"
-        subtitle: "TMP"
+        title: "The certificate does not appear to \nexist: contact support"
+        subtitle: "You may be asked for this data to investigate the problem:"
+        cta: "Contact support"
       notOperational:
         title: "The european COVID certificate is not currently operational"
         subtitle: "TMP"
       temporarilyNotAvailable:
         title: "The service is temporarily unavailable"
         subtitle: "Please try again later"
       wrongFormat:
-        title: "TMP"
-        subtitle: "TMP"
+        title: "This certificate cannot be \nrecovered: contact support"
```

Comment:
```suggestion
        title: "This certificate cannot be \nretrieved: contact support"
```

Reply:
Here the result:
<img src="https://user-images.githubusercontent.com/11773070/120995807-106c2500-c786-11eb-9c5d-e7242665aa1d.png" width=300>